pynicotine/gtkgui/userbrowse.py

```diff
@@ -930,9 +930,9 @@ def on_download_directory_to_selected(self, selected, recurse):
     def on_download_directory_to(self, *args, recurse=False):
 
         if recurse:
-            str_title = _("Select Destination for Downloading Folder with Subfolders from User")
+            str_title = _("Select destination folder for this recursive download")
```

Comment:
This is a very tricky one isn't it. I tried several things when I was writing these strings and nothing seems to make sense.

I am concerned that folks won't know what "recursive" means, this is not a very common English word. My intention was to make it clear that Sub Folders will be included in the download, it is hard to explain clearly.

Reply:
It is indeed. Even more so if you want to take translatability into account.

The word "recursive" can be found 37 times in ./pynicotine/ and it's not that uncommon in English. Even the simple english wikipedia explains it.

Reply:
Us and many users are all geeks so this technical language is understood by most, but not everybody understands the purpose of this function. It does cause confusion so I feel mats choice is correct in https://github.com/nicotine-plus/nicotine-plus/commit/9ad9389b9a4a0090577155330c41d267bfdf4b9e and the rest of the interface needs to follow suit.

I don't think it will ever be likely that the number/size of selected Subfolders and Files will be counted by this function, so this string is probably going to be fairly permanent.

Reply:
In the meanwhile, @mathiascode has replaced `recursive` with a construction containing something like `including subfolders` and I adjusted the PR to reflect that change.